embassy-stm32-wpan/src/evt.rs

```diff
@@ -105,6 +106,14 @@ impl EvtBox {
         Self { ptr }
     }
 
+    pub fn evt<'a>(&self) -> &'a [u8] {
```

Comment:
This is unsound. It allows you to hold on to the slice after you've dropped the packet.

You probably want `pub fn evt<'a>(&'a self) -> &'a [u8]`.
Or `pub fn evt(&self) -> &[u8]` which is the same using lifetime elision.

Reply:
oh this is unused, forgot to remove it